avocado/core/loader.py

```diff
@@ -893,18 +892,22 @@ def _make_existing_file_tests(self, test_path, make_broken,
             if os.access(test_path, os.X_OK):
                 # Module can't be imported, and it's executable. Let's try to
                 # execute it.
-                return self._make_test(test.SimpleTest, test_path)
+                return self._make_test(test.SimpleTest, test_path,
+                                       subtests_filter=subtests_filter)
             else:
                 return make_broken(NotATest, test_path, self.__not_test_str)
 
     @staticmethod
-    def _make_test(klass, uid, description=None):
+    def _make_test(klass, uid, description=None, subtests_filter=None):
         """
         Create test template
         :param klass: test class
         :param uid: test uid (by default used as id and name)
         :param description: Description appended to "uid" (for listing purpose)
```

Comment:
For consistency, let's add this new parameter to the docstring.

Reply:
yes, sure